src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -120,7 +120,7 @@ export const LongButtonText: StoryObj<Args> = {
   },
   parameters: {
     // Is flaky on chromatic, disabling for now
-    chromatic: { delay: 300 },
+    chromatic: { disableSnapshot: true },
```

Comment:
> now I'm also wondering what we even gain from this snapshot because it doesn't even show the tooltip

that's the problem, right? that it shows up in Storybook but for some reason not in Chromatic 🤔  fine with disabling for now though